packages/notifi-dapp-example/src/components/Toggle.tsx

```diff
@@ -23,7 +23,9 @@ export const Toggle: React.FC<ToggleProps> = ({
         className="sr-only peer"
         checked={checked}
         disabled={disabled}
-        onChange={onChange}
+        onChange={() => {
+          onChange?.();
```

Comment:
Why do we need to use optional chaining here?

Reply:
since `onChange` is optional in props, so It's safe to use optional chaining here.